python/pyproject.toml

```
@@ -123,3 +119,21 @@ wheel.packages = ["rmm"]
 provider = "scikit_build_core.metadata.regex"
 input = "rmm/VERSION"
 regex = "(?P<value>.*)"
+
+[tool.rapids_builder]
+build-backend = "scikit_build_core.build"
+commit-file = "rmm/_version.py"
```

With the [new commit handling](https://github.com/rapidsai/rapids-build-backend/pull/3), this should be:

```suggestion
commit-file = "rmm/GIT_COMMIT"
```